client/app/components/composites/ManageAvailabilityHeader/ManageAvailabilityHeader.css

```diff
@@ -0,0 +1,35 @@
+.root {
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  overflow: hidden;
+}
+
+.colorLayer {
```

Comment:
FYI @jannekoivistoinen used a box shadow inset cool trick to add a color overlay without extra divs. We are using it already on the [success stories page hover](https://www.sharetribe.com/stories.html):

![screen shot 2016-11-25 at 16 03 37](https://cloud.githubusercontent.com/assets/429876/20627510/2d279380-b329-11e6-87d7-0f3699f4e047.png)

Would that we useful here?